config/locales/en.yml

```diff
@@ -21,12 +21,14 @@ en:
     has_many_new: "Add New %{model}"
     has_many_delete: "Delete"
     has_many_remove: "Remove"
+    move: "MOVE"
     filters:
       buttons:
         filter: "Filter"
         clear: "Clear Filters"
       predicates:
         contains: "Contains"
+        eq: "equals"
```

Comment:
Not sure but I think this a "native" ransack predicate so that's why we don't provide translation. It should be translated in ransack? @Fivell?

Reply:
You are right, "eq" is in Ransack translations [ransack_en](https://github.com/activerecord-hackery/ransack/blob/master/lib/ransack/locale/en.yml). I could make a PR to Ransack for italian lang